guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -532,7 +532,77 @@ class View extends Template
 
 ### 6.4. Service Contracts (Application) layer
 
-We are reviewing this section and will publish it soon.
+6.4.1. Location of API interfaces
+
+6.4.1.1. Service contract interfaces SHOULD be placed in separate API modules. All other modules will depend on the API module, while implementations could be easily swapped via `di.xml`. API modules must have `Api` suffix in their names. For example, if a module is named `MyModule`, its APIs SHOULD be declared in a module named `MyModuleApi`.
```

Comment:
replace with: Other modules will depend upon the API module, and implementations can be easily swapped using `di.xml`. API modules must contain the `API` suffix in their names.